src/lsp/cobol_ptree/terms_visitor.ml

```diff
@@ -286,6 +337,10 @@ and fold_expr (v: _ #folder) =
 and fold_expr' (v: _ #folder) =
   handle' v#fold_expr' ~fold:fold_expr v
 
+and fold_leading_trailing (_: _ #folder) : leading_trailing -> 'a -> 'a = function
```

Comment:
Here you don't use the `fold_leading_trailing` method: `= leaf v#fold_leading_trailing` should be sufficient.